python_scripts/ensemble_gradient_boosting.py

```diff
@@ -136,7 +136,7 @@ def generate_data(n_samples=50):
 # select this sample in `data_train`.
 
 # %%
-data_max = data_train.iloc[-2, 0]
+data_max = data_train.iloc[[-2]]
```

Comment:
While you are at it,  maybe you can change `data_max` -> `sample` because `data_max` is a weird name.

Reply:
Done, thanks!